src/config.rs

```diff
@@ -9,6 +9,11 @@ pub struct Element {
     pub exif_tags: Vec<String>,
     /// Optional regex pattern and its replacement
     pub replace: Option<Vec<Replacement>>,
+    /// Optional from and to cases
+    /// from
+    pub case_from: Option<String>,
+    /// to
+    pub case_to: Option<String>,
```

Comment:
I think I would structure it a little different. Instead of having the two case-fields as part of the `Element` struct, I'dd extract them to different struct:

```rust
pub struct CaseConversion {
    from: String,
    to: String,
}

// in `Element`:
   pub case_conversion: Option<CaseConversion>,
```